backend/lib/spark-api-deal-submitter.js

```diff
@@ -0,0 +1,127 @@
+/** @import {PgPool, Queryable} from '@filecoin-station/deal-observer-db' */
+import Cursor from 'pg-cursor'
+
+/**
+ * Finds deals that haven't been submitted to the Spark API yet and submits them.
+ *
+ * @param {PgPool} pgPool
+ * @param {number} batchSize
+ * @param {(eligibleDeals: Array) => Promise<void>} submitDeals
+ */
+export const findAndSubmitUnsubmittedDeals = async (pgPool, batchSize, submitDeals) => {
+  console.debug(`Finding and submitting deals using batchSize: ${batchSize}`)
+  for await (const unsubmittedDeals of findUnsubmittedDeals(pgPool, batchSize)) {
+    console.debug(`Found ${unsubmittedDeals.length} unsubmitted deals`)
+    try {
+      const formattedDeals = formatUnsubmittedDealsForSparkApi(unsubmittedDeals)
+      await submitDeals(formattedDeals)
+      console.debug(`Successfully submitted ${formattedDeals.length} deals`)
+      await markDealsAsSubmitted(pgPool, unsubmittedDeals)
+    } catch (e) {
+      console.error('Failed to submit deals:', e)
+    }
+  }
+}
+
+/**
+ * Finds deals that haven't been submitted to spark api.
+ * For deals to be submitted they must
+ * - have not been submitted yet
+ * - were created more than 2 days ago
```

Comment:
Can you please briefly explain why we submit only days older than 2 days and/or link to the GitHub comment explaining the details.